doc/conf.py

```diff
@@ -137,7 +137,8 @@
             }
         ],
         "use_edit_page_button": True,
-        "gitter_room": "spyder-ide/public"
+        "gitter_room": "spyder-ide/public",
+        "alert_message": "This documentation is still a work in progress!"
```

Comment:
```suggestion
        "alert_message": "This documentation is still a work in progress. Check out the [Spyder 3 docs](https://docs.spyder-ide.org) for the stable version"
```

I suggest this message instead so people can access to our current docs.